go/vt/sqlparser/comments.go

```diff
@@ -32,6 +32,8 @@ const (
 	DirectiveQueryTimeout = "QUERY_TIMEOUT_MS"
 	// DirectiveScatterErrorsAsWarnings enables partial success scatter select queries
 	DirectiveScatterErrorsAsWarnings = "SCATTER_ERRORS_AS_WARNINGS"
+	// DirectiveMaxPayloadSize sets a maximum payload size in bytes. Only supported for INSERTS.
+	DirectiveMaxPayloadSize = "MAX_PAYLOAD_SIZE"
```

Comment:
I've been wondering if this feature makes more sense as a flag for the entire environment rather than a per query comment, like the gRPC max packet size: https://github.com/tinyspeck/vitess/blob/master/go/vt/grpccommon/options.go#L32

Reply:
+1